### ISSUE TITLE
Radix dialog missing aria-describedby warning

### ISSUE THREAD

**Issue by shahwali-dev** [archestra-ai/archestra#5008]
Hi, when opening the dialog component, radix UI shows a warning in browser console about missing aria-describedby. 

I attached a video below to show the warning.

To fix this, we need to change the dialog content component in this file:
/archestra/platform/frontend/src/components/ui/dialog.tsx

We just need to pass aria-describedby={undefined} to DialogContent to stop this console warning when we don't need description. 

I tested it locally and warning is gone. Creating a PR now.

https://github.com/user-attachments/assets/655cba7a-6c19-423f-a4b0-488d5ec08c3d

---

<a href="https://archestra.ai/contributor-onboard" rel="nofollow noreferrer noopener" target="_blank">

<img alt="Archestra Contributor" src="https://raw.githubusercontent.com/archestra-ai/archestra/main/docs/assets/archestra-contributor-banner.webp"/>

</a>
